src/Resources/ResourceManager/Entities/Policy/EnforcementMode.cs

```diff
@@ -0,0 +1,32 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Entities.Policy
+{
+    /// <summary>
+    /// The policy definition mode.
+    /// </summary>
+    public class EnforcementMode
```

Comment:
Any reason you preferred using a class with const string vs. enum? I think the help generation will be better with an enum.

Reply:
Changed it to enum now.